cura/PrinterOutput/PrinterOutputModel.py

```diff
@@ -73,6 +82,15 @@ def updateType(self, type):
             self._type = type
             self.typeChanged.emit()
 
+    @pyqtProperty(str, notify = buildplateChanged)
+    def buildplate(self):
+        return self._buildplate_name
+
+    def updateBuildplate(self, buildplate_name):
```

Comment:
Naming is a bit vague, maybe something like setBuildplateName?

Reply:
I rename it in dfb903fb8113b2694085d95f80d3982ab4707a16